packages/mobile/src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -0,0 +1,193 @@
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Spacing } from '@celo/react-components/styles/styles'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useLayoutEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import * as Keychain from 'react-native-keychain'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
+import { setUseBiometry } from 'src/app/actions'
+import { supportedBiometryTypeSelector } from 'src/app/selectors'
+import { HeaderTitleWithSubtitle, nuxNavigationOptions } from 'src/navigator/Headers'
+import { navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarTextButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import useRegistrationStep from 'src/onboarding/registration/useRegistrationStep'
+import { default as useSelector } from 'src/redux/useSelector'
+import Svg, { Path } from 'svgs'
+
+type Props = StackScreenProps<StackParamList, Screens.EnableBiometry>
+
+const biometryImageMap: { [key in Keychain.BIOMETRY_TYPE]: JSX.Element } = {
+  [Keychain.BIOMETRY_TYPE.FACE_ID]: <FaceIDImage />,
+  [Keychain.BIOMETRY_TYPE.TOUCH_ID]: <TouchIDImage />,
+  [Keychain.BIOMETRY_TYPE.FINGERPRINT]: <FingerprintImage />,
+  [Keychain.BIOMETRY_TYPE.FACE]: <FaceImage />,
+  [Keychain.BIOMETRY_TYPE.IRIS]: <FaceImage />,
```

Comment:
i didn't pay too much attention to these for now, since this experiement is for iOS users (fingerprint/face/iris is for android)